daemon/src/maker_cfd.rs

```diff
@@ -513,20 +497,7 @@ impl Actor {
             }
         };
 
-        // 2. check if order has acceptable amounts
-        if quantity < current_order.min_quantity || quantity > current_order.max_quantity {
-            tracing::warn!(
-                "Order rejected because quantity {} was out of bounds. It was either <{} or >{}",
-                quantity,
-                current_order.min_quantity,
-                current_order.max_quantity
-            );
-
-            self.reject_order(taker_id, order_id, conn).await?;
-            return Ok(());
-        }
```

Comment:
@da-kami @bonomat I had to move this down because one of my refactorings changes the function signature of `append_cfd_state` to take the entire `Cfd`. Thus we need to have one available here to pass it down.

I think that actually uncovered a potential problem: I think calling `reject_order` in there never worked because we would not have had a `Cfd` in the database that we could append the state to.